x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/new_index/select_connector/select_connector.tsx

```diff
@@ -77,20 +91,69 @@ export const SelectConnector: React.FC = () => {
         <EuiFormFieldset
           legend={{
             children: (
-              <EuiText color="subdued" size="s">
-                <p />
-              </EuiText>
+              <EuiCallOut
+                size="m"
+                title={i18n.translate(
+                  'xpack.enterpriseSearch.content.indices.selectConnector.callout.title',
+                  { defaultMessage: 'Elastic connectors' }
+                )}
+                iconType="iInCircle"
+              >
+                <p>
+                  <FormattedMessage
+                    id="xpack.enterpriseSearch.content.indices.selectConnector.description.textcloud"
+                    defaultMessage="Use {native} connectors out of the box on Elastic cloud. {learnMore}"
+                    values={{
+                      learnMore: (
+                        <EuiLink target="_blank" href={docLinks.connectorsNative}>
+                          {LEARN_MORE_LINK}
+                        </EuiLink>
+                      ),
+                      native: (
+                        <b>
+                          {i18n.translate(
+                            'xpack.enterpriseSearch.content.indices.selectConnector.callout.description.native',
+                            { defaultMessage: 'native' }
+                          )}
+                        </b>
+                      ),
+                    }}
+                  />
+                  <br />
+                  <br />
+                  <FormattedMessage
+                    id="xpack.enterpriseSearch.content.indices.selectConnector.description.selfManaged.text"
+                    defaultMessage="For advanced use cases, {connectorsClient} connectors on your own infrastructure, customize existing connectors, or build your own using our connector framework and clients. {learnMore}"
```

Comment:
Text should read:

`For advanced use cases, deploy connector clients on your own infrastructure, to customize existing connectors, or build your own using our connector framework and clients. `

Reply:
![Screenshot 2023-05-10 at 19 39 31](https://github.com/elastic/kibana/assets/1410658/4e7ca5db-208f-498f-aeac-c37bd155ce8a)
Changed, pushing now

Reply:
Sorry there's some duplicate comments, GitHub API is buggy 😄

Reply:
Should be "connector clients" there (missing an "s") 🙏

Reply:
I took screenshot before adding that sorry :D it is in